api/v1alpha1/mirrorpeer_types.go

```diff
@@ -61,10 +61,6 @@ type MirrorPeerSpec struct {
 	// +kubebuilder:validation:MinItems=2
 	Items []PeerRef `json:"items"`
 
-	// SchedulingIntervals is a list of intervals at which mirroring snapshots are taken
-	// +kubebuilder:validation:Optional
-	SchedulingIntervals []string `json:"schedulingIntervals,omitempty"`
```

Comment:
If you remove it, what happens to clusters that already have older MirrorPeer CR? How will upgrades work?
We need to handle all cases. We can't add a breaking change in minor release.

Reply:
Do we just upgrade the API version and add a conversion webhook for the process ?

Reply:
That's generally the process for major releases. For minor releases you can just retain the field and add a comment that says it is deprecated.

Reply:
done